### DISCUSSION TITLE
Tell us about yourself, your goals!

### DISCUSSION THREAD

**Discussion by nitya** [microsoft/30daysof#3] [Announcements]
## 👋 Welcome, Learners!!

This is the Discussion forum for the [30DaysOf](https://aka.ms/30DaysOf) learning initiative. We'll use this forum to share announcements on upcoming #30Days roadmaps, relevant learning events and curricula, and ways for you to showcase your insights and achievements with community!

We hope you use this to 
 * Discover and connect with other learners
 * Pick a #30Days learning goal and build a daily habit of learning-by-doing
 * Help others by sharing your knowledge with insights, open-source projects and articles
 * Treat every other learner with empathy and understanding - our lived experiences will differ!

Start off by introducing yourself!! Here are a few suggestions if useful:
 * Your name and pronouns - how you want to be addressed in conversations
 * Your region - connect with other learners in the same city, state or region.
 * Your interests - what technologies are you working with - or learned recently.
 * Your learning goal - what are you planning to learn next?
 * Your expectations - what are you looking for in a learning community?

**Comment by nshah1729** [comment]
Hey there, I'm Nishant Shah(he/him) trying to learn Web Development. Thanks to @nitya @microsoftopensource who has started this amazing initiative to create a habit(which is hard🥲) of coding. I live in West Bengal, India pursuing my engineering at Jadavpur University. I will be learning Full Stack Dev over the next 30 Days. I'll be starting with making some basic projects with vanilla CSS and after some brushing up, I will start learning ReactJS😍 . For each project, I will also add the backend to it using NodeJS, and MongoDB.
I'm looking forward to a great learning experience✌️

**Reply by nshah1729** [reply]
Also wanted to know the best resources to learn ReactJS from @nitya @alfredodeza @leestott

**Reply by nitya** [reply]
Hi @nshah1729 -- welcome to the site!! Love the idea of a #30Days of Web development. Definitely check out the Curricula on the site - the web dev one is project-driven which is good. There are a number of good ReactJS resources out there.

- Always good to start with the docs. In case you missed it, there is a [beta version of the React docs here](https://beta.reactjs.org/learn) which does a good job of explaining concepts with interactive examples and visuals. The site is in-progress (70% content ready for "Learn React) but I find it useful.
- I also like the [Reactjsexample.com](https://reactjsexample.com/) site because it lets you look at others' code repos for good app examples using different front-end and back-end technologies with React. Study them to learn how things work and then clone/extend and contribute back.
- My third resource is [patterns.dev](https://www.patterns.dev/) - a great site maintained by Addy Osmani with example code and great animations to help you learn visually. Patterns work for vanilla JS and React apps.

Hope that helped!

**Comment by maco-eng** [comment]
Hi, I am Gbaglo simeon( he/him) from lagos ,Nigeria, area of interest are data science and machine learning
